data/json/items/gunmod/sights.json

```diff
@@ -66,7 +66,7 @@
     "type": "GUNMOD",
     "name": { "str": "holographic sight" },
     "//": "EOTech XPS3",
-    "description": "Used extensively by military forces, it increases accuracy and weight.  This is a step up from a red dot sight.",
+    "description": "An advanced firearm optic with an adjustable circular hologram reticle imposed on the field of view.  Used extensively by military forces, it increases accuracy and weight.  This is a step up from a red dot sight.",
```

Comment:
I dont feel like "imposed" really works here

Reply:
Try 'superimposed over the field of view.'